### PR TITLE
fix: LEAP-889: remove env_vars from context log

### PR DESCRIPTION
**Pull request by sajarin**
### PR fulfills these requirements
- [X] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [X] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [X] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [X] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [X] Backend (Database)
- [ ] Backend (API)
- [ ] Frontend

### Describe the reason for change

Based on the current code, context logs currently collect environment variables that are prefixed with `LABEL_STUDIO`. There may be a case where someone creates a non label studio related env variable and that accidentally gets exposed to us. This PR is meant to remove env_vars from the payload entirely. 

#### What does this fix?
This removes the env_var object from the payload and also removes the get_label_studio_env function (as this is no longer needed). 

#### What is the new behavior?
The new behavior is that environmental variables are no longer present in our context logs. 

#### Does this change affect security?
Yes, this minimizes our exposure to sensitive user data. 

### Does this PR introduce a breaking change?
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
- [X] unit

### CODE REVIEW

File: label_studio/feature_flags.json [hunk 1]
```diff
@@ -12,10 +12,7 @@
         "variation": 0
```
Comment (jombooth) [inline label_studio/feature_flags.json]: Can we revert the changes in this file? Looks like it was autoformatted

File: label_studio/tests/test_contextlog.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+"""This file and its contents are licensed under the Apache License 2.0. Please see the included NOTICE for copyright information and LICENSE for a copy of the license.
+"""
```
Comment (jombooth) [inline label_studio/tests/test_contextlog.py]:
```suggestion
```

File: Dockerfile [hunk 3]
```diff
@@ -37,6 +37,7 @@ ENV DEBIAN_FRONTEND=noninteractive \
     POETRY_VIRTUALENVS_CREATE=false \
     DJANGO_SETTINGS_MODULE=core.settings.label_studio \
     LABEL_STUDIO_BASE_DATA_DIR=/label-studio/data \
+    LABEL_STUDIO_TEST_ENV_VAR=true \
```
Comment (jombooth) [inline Dockerfile]:
```suggestion
```

File: label_studio/tests/test_contextlog.py [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+import pytest
+import responses
+from django.test import override_settings
+
+
+@responses.activate
+@pytest.mark.django_db
+@override_settings(CONTEXTLOG_SYNC=True)
+def test_contextlog(business_client):
```
Comment (jombooth) [inline label_studio/tests/test_contextlog.py]:
```suggestion


@responses.activate
@pytest.mark.django_db
def test_contextlog(business_client, settings):
    settings.CONTEXTLOG_SYNC = True
```

I realized that we could leverage pytest's `settings` fixture instead to make this a little simpler

File: label_studio/tests/test_contextlog.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
+"""This file and its contents are licensed under the Apache License 2.0. Please see the
+included NOTICE for copyright information and LICENSE for a copy of the license.
+"""
```
Comment (jombooth) [inline label_studio/tests/test_contextlog.py]: Ah I was actually proposing we remove the license string, I don't think these are needed

File: label_studio/core/settings/base.py [hunk 6]
```diff
@@ -739,3 +739,7 @@ def collect_versions_dummy(**kwargs):
 
 CLOUD_STORAGE_CHECK_FOR_RECORDS_PAGE_SIZE = get_env('CLOUD_STORAGE_CHECK_FOR_RECORDS_PAGE_SIZE', 10000)
 CLOUD_STORAGE_CHECK_FOR_RECORDS_TIMEOUT = get_env('CLOUD_STORAGE_CHECK_FOR_RECORDS_TIMEOUT', 60)
+
+CONTEXTLOG_SYNC = False
```
Comment (wesleylima) [inline label_studio/core/settings/base.py:743]: Do we want to `get_bool_env` for this one too? Or is it not supposed to be settable through env variables? How would this be set then?

Comment (sajarin) [reply label_studio/core/settings/base.py:743]: This variable shouldn't really be settable, we specify it here in the settings to that we can override it via the settings fixture in conftest.

**Review by wesleylima — APPROVED** [review]

**Review by jombooth — APPROVED** [review]
LGTM, nice work!
